packages/components/src/components/@deprecated/input/controller.ts

```diff
@@ -124,6 +127,12 @@ export class InputController extends ControlledInputController implements Watche
 	}
 
 	public validateMsg(value?: Stringified<MsgPropType>): void {
+		if (value === undefined) {
+			value = {
```

Comment:
@deleonio Ich gehe davon aus, dass das so funktioniert.

Man könnte bloß argumentieren, dass es etwas unschön ist, dass "keine Fehlermeldung" bzw. ein `undefined`-Value intern jetzt in ein Objekt vom Typ Error übersetzt werden. Sauberer fände ich es schon, wenn wir "kein Fehler" auch konsistent als `undefined` repräsentieren könnten.

An anderer Stelle haben wir deshalb die Validierung mit `defaultValue: undefined` angestoßen. Wäre das auch hier eine Option?

Reply:
defaultValue hatte ich gesehen, aber so schlau dabei an undefined zu denken war ich nicht.

Meine Hauptergänzung war eigentlich _error zu berücksichtigen, dass das da auch sauber funktioniert.

Dann müssen wir das nochmal ändern und dann nschtesten. So wie jetzt funktioniert es zwar, ist aber sicher nicht optimal.

Reply:
Done: https://github.com/public-ui/kolibri/pull/7113/files